hicommonwealth/commonwealth
# Fix upvote persistency

## Link to Issue
Closes: #8185
Closes: #8184  

## Description of Changes
8185:
- changed `vote_weight` into `voting_weight` as this must have been a typo
- from the code review, `vote_weight` is used for community stake and `voting_weight` is for reactions

8184:
- sometimes data was missing name or avatar url
- resolved it by using User component to display avatar + username

## Test Plan
8185:
- buy stake
- create a thread
- upvote it 
- goto thread page/come back OR refresh
- the number should be persisted

8184: 
- open drawer with upvotes
- make sure that all names and avatars are visible

## Deployment Plan

1. n/a
